Apps/WebClient/src/ClientApp/src/views/dependents.vue

```diff
@@ -84,7 +86,7 @@ export default class DependentsView extends Vue {
 }
 </script>
 <template>
-    <div class="m-3">
+    <div class="m-3 flex-grow-1 d-flex flex-column">
```

Comment:
Q - Why is this necessary? the resource center can float without having to interact with the rest of the layoutt

Reply:
The resource centre is sticky rather than abolutely positioned.  It's located at the bottom of the page content.  If the parent is not tall enough, the resource centre will not appear in the bottom right - it will appear in the middle-right where the parent content ends.

The .flex-grow allows the parent to grow to fill the remaining space that is provided by _its_ parent's .fill-height.  Applying .d-flex to it then lets us position the resource centre at the bottom of it using .mt-auto.